ruby/vampires.rb

```diff
@@ -1,3 +1,5 @@
+#random comment so I can make a pull request
+
 puts "How many employees will be processed today?"
 new_employees = gets.chomp.to_i
 

```

Comment:
@Joel-B-Williams I see a lot good things here.  You're using document control in a meaningful way, showing a base understanding for a conditional loop, and tried to encapsulate your code in a method!

Lets talk about some of the ordering logic.
```
if emp_name == "Tu Fang" || emp_name == "Drake Cula"
  is_vampire = "Definitely a vampire."
end
```

In this case if the name is Fang or Drake then we know this person is a vampire so logically this would be the first thing you should consider checking.  Is there any reason to check anymore of the code if we know they are a vampire already?

Another refactor you could try is with the age calculation.
```
age_right = (2016 - age) == birth_year
```
Look at the time class in the Ruby docs to fine `Time.now.year` and see if you can find a dynamic way to add this to your code to make sure that we don't have to rewrite this file every year.